imnodes.cpp

```diff
@@ -631,14 +631,35 @@ void BeginCanvasInteraction(ImNodesEditorContext& editor)
 
     const bool started_panning = GImNodes->AltMouseClicked;
 
-    if (started_panning)
+    // Handle mini-map interactions
+    if (ImGui::IsWindowFocused() and
```

Comment:
When the window is out of focus, it is possible to start a box selection inside the minimap, due to the start of the box selection interaction not being behind a window focus-check. So I think we either need to remove the `ImGui::IsWindowFocused()` check here, or early-out of the function if the window is not being hovered.

ImGui doesn't prevent UI element interaction when the window is out of focus, so I think for interaction consistency it might make sense to just not have the focus check here, letting the window come in to focus when starting the interaction.

Reply:
Sounds reasonable to me. Will remove.